types/cesium/index.d.ts

```diff
@@ -3905,52 +3916,109 @@ declare namespace Cesium {
     }
 
     class Model {
-        show: boolean;
-        modelMatrix: Matrix4;
-        scale: number;
-        minimumPixelSize: number;
-        id: any;
         activeAnimations: ModelAnimationCollection;
+        readonly allowPicking: boolean;
+        readonly asynchronous: boolean;
+        readonly basePath: string;
+        readonly boundingSphere: BoundingSphere;
+        clampAnimations: boolean;
+        clippingPlanes: ClippingPlaneCollection;
+        color: Color;
+        colorBlendAmount: number;
+        colorBlendMode: ColorBlendMode;
+        credit: Credit;
         debugShowBoundingVolume: boolean;
         debugWireframe: boolean;
-        gltf: any;
-        basePath: string;
-        boundingSphere: BoundingSphere;
-        ready: boolean;
-        readyPromise: Promise<Model>;
-        asynchronous: boolean;
-        allowPicking: boolean;
+        distanceDisplayCondition: DistanceDisplayCondition;
+        readonly gltf: any;
+        heightReference: HeightReference;
+        id: any;
+        imageBasedLightingFactor: Cartesian2;
+        readonly incrementallyLoadTextures: boolean;
+        lightColor: Cartesian3;
+        luminanceAtZenith: number;
+        maximumScale: number;
+        minimumPixelSize: number;
+        modelMatrix: Matrix4;
+        readonly pendingTextureLoads: number;
+        readonly ready: boolean;
+        readonly readyPromise: Promise<Model>;
+        scale: number;
+        show: boolean;
```

Comment:
missing `shadows` field: https://cesium.com/docs/cesiumjs-ref-doc/Model.html?classFilter=model#shadows

Reply:
strange that it's not in the constructor - https://cesium.com/docs/cesiumjs-ref-doc/Model.html?classFilter=model#Model